### PR TITLE
Fix live example in Other form controls

### LINKED ISSUE #9421: Issue with "Other form controls": autocomplete box example is showing select box instead.
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/Other_form_controls

#### What information was incorrect, unhelpful, or incomplete?
In the Autocomplete box example, the HTML code example is accurate but the iFrame seems to be showing a select box, instead of an autocomplete. 

#### Specific section or headline?
Autocomplete box, live basic example. HTML code line 1581.

#### What did you expect to see?
Expected an input field, where I could type text and suggestions would show up as a drop down box. 

#### Did you test this? If so, how?
I copied the basic example HTML code into my own file and it worked as expected. I suspect that the issue is with the iFrame's link for this example.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/forms/other_form_controls`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Forms/Other_form_controls
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/forms/other_form_controls/index.html
* Last commit: https://github.com/mdn/content/commit/d4e436594b6c72d99e9b796235e46c6c7c6b6c6e
* Document last modified: 2021-09-30T23:28:46.000Z

</details>

### PR DESCRIPTION
**Pull request by digi-booster**
fix #9421

#### Summary

- increase Multi-line text fields example height to remove vertical scrolling
- autocomplete live example fix
- typo fix

#### Related issues

fixes #9421 
#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/forms/other_form_controls/index.html [hunk 1]
```diff
@@ -305,7 +305,7 @@ <h2 id="Test_your_skills!">Test your skills!</h2>
 
 <h2 id="Summary">Summary</h2>
 
-<p>As you'll have seen in the last few articles, there are a lot of different types of available form element. You don't need to remember all of these details at once, and can return to these articles as often as you like to check up on details.</p>
+<p>As you'll have seen in the last few articles, there are a lot of different types of form elements are available. You don't need to remember all of these details at once, and can return to these articles as often as you like to check up on details.</p>
```
Comment (wbamberg) [inline files/en-us/learn/forms/other_form_controls/index.html:308]:
```suggestion
<p>As you'll have seen in the last few articles, there are many different types of form control. You don't need to remember all of these details at once, and can return to these articles as often as you like to check up on details.</p>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for this PR, @digi-booster !

The issue here is that there are two headings with the same text: "Basic example". When this happens Yari adds a numeric suffix to the IDs for the additional headings, so they are unique.

So the first "Basic example" is "Basic_example" and the second is "Basic_example_2".

So the fix here should not be to change the heading text: instead change the argument that gets passed into the `EmbedLiveSample` call:

```
 <p>{{EmbedLiveSample("Basic_example_2", 120, 120)}}</p>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for the update @digi-booster !

This fixed the editorial suggestion but not the comment in https://github.com/mdn/content/pull/9423#pullrequestreview-769171951 .

File: files/en-us/learn/forms/other_form_controls/index.html [hunk 2]
```diff
@@ -192,7 +192,7 @@ <h4>Basic example</h4>
   &lt;option&gt;Pear&lt;/option&gt;
 &lt;/datalist&gt;</pre>
 
-<p>{{EmbedLiveSample("Basic_example", 120, 120)}}</p>
+<p>{{EmbedLiveSample("Autocomplete_basic_example", 120, 120)}}</p>
```
Comment (digi-booster) [inline files/en-us/learn/forms/other_form_controls/index.html:195]:
```suggestion
 <p>{{EmbedLiveSample("Basic_example_2", 120, 120)}}</p>
```

**Comment by digi-booster** [comment]
> Thanks for the update @digi-booster !
> 
> This fixed the editorial suggestion but not the comment in [#9423 (review)](https://github.com/mdn/content/pull/9423#pullrequestreview-769171951) .

Thanks for the help @wbamberg, but I think [live samples](https://developer.mozilla.org/en-US/docs/MDN/Structures/Live_samples) don't cover what to do if the same type of title is written more than once, so I thought maybe this would fix it.

File: files/en-us/learn/forms/other_form_controls/index.html [hunk 3]
```diff
@@ -175,7 +175,7 @@ <h3 id="Autocomplete_box">Autocomplete box</h3>
 
 <p>Once a data list is affiliated with a form widget, its options are used to auto-complete text entered by the user; typically, this is presented to the user as a drop-down box listing possible matches for what they've typed into the input.</p>
 
-<h4>Basic example</h4>
+<h4>Autocomplete Basic example</h4>
```
Comment (digi-booster) [inline files/en-us/learn/forms/other_form_controls/index.html:178]:
```suggestion
<h4>Basic example</h4>
```

**Review by wbamberg — APPROVED** [review]
Thanks @digi-booster , this looks great now.

> I think live samples don't cover what to do if the same type of title is written more than once

Yeah, you're right. Unfortunately this gets a bit more confusing in Markdown, because you don't see the heading ID any more: it's always generated from the text. So you have to know how the heading generation works (so we should document it better).

**Pull request merged**